runtime/bcutil/cfreader.c

```diff
@@ -1168,25 +1168,28 @@ readPool(J9CfrClassFile* classfile, U_8* data, U_8* dataEnd, U_8* segment, U_8*
 
 		case CFR_CONSTANT_Module:
 		case CFR_CONSTANT_Package:
-			if (classfile->majorVersion < 53) {
-				errorCode = J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V53__ID;
-				offset = (U_32)(index - data - 1);
-				goto _errorFound;
-			}
+			if (vmVersionShifted >= BCT_Java9MajorVersionShifted) {
```

Comment:
This version check shouldn't be necessary - an unknown constant should result in a ClassFormatError.  This would just give it a different message.